docs/subnets/create-first-subnet.md

```diff
@@ -0,0 +1,208 @@
+# Build Your First Subnet
+
+The first step of learning Subnet development is learning to use [Avalanche-CLI](https://github.com/ava-labs/avalanche-cli).
+
+The best way to get started is by jumping in and deploying your first Subnet.
+
+This tutorial will walk you through the process of using Avalanche-CLI for the first time by creating a Subnet,
+deploying it to the local network, and connecting to it with Metamask.
```

Comment:
I wonder if we should more aggressively mention Core here instead of Metamask. It would make sense IMO.

Reply:
They are aware and will add support eventually I believe, I've mentioned it a few times

Reply:
Created a ticket to update this eventually https://github.com/orgs/ava-labs/projects/3/views/8#:~:text=Add%20Core%20to%20Subnet%20tutorials%23482